applications/jupyter/README.md

```diff
@@ -171,6 +137,6 @@ We allow user to set their own domains, in the `variables.tf` file. Since we are
 
```

Comment:
What does this mean (the ingress thing)?

Reply:
Updated statement